### PR TITLE
Add documentation for supported networks and Clients

### PR DESCRIPTION
**Pull request by stdevMac**
## Changes:
- Added documentation for supported networks and clients on each of those networks

## Types of changes
- [x] Documentation Update

## Testing
**Requires testing**

- [ ] Yes
- [x] No

**In case you checked yes, did you write tests?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Changes requested. Good idea. If Besu and Erigon PRs are merged before this, then they must be added to the proper sections in the new pages.

File: docs/docs/networks/goerli.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+id: goerli
+---
+
+# Goerli
+
+Goerli is an Ethereum test network that allows for blockchain development testing before deployment on Mainnet, the
+main Ethereum network. The Proof-of-Authority test network was established in March 2019. It uses the Clique consensus
+mechanism and was originally proposed by Chainsafe and Afri Schoedon.
+
+## Supported Consensus Clients
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/goerli.mdx:13]: done

File: docs/docs/networks/sepolia.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 3
+id: sepolia
+---
+
+# Sepolia
+
+Sepolia is a proof-of-work (PoW) testnet where dApp developers can test their applications. What differentiates Sepolia
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/sepolia.mdx]: done

File: docs/docs/networks/mainnet.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+For running a full node (without a validator node) with random clients, you only need to add the `--no-validator` to `sedge cli`. For example:
+
+```bash
+sedge cli --no-validator
+```
+
+:::info
+
+Sedge uses Mainnet as default network, so you don't have to specify.
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/mainnet.mdx]: done

File: docs/docs/networks/mainnet.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 1
+id: mainnet
+---
+
+# Mainnet
+
+Mainnet is the public and decentralized network of Ethereum. It is the network that all validators participate in to
+earn staking rewards and is the network that all users interact with to send transactions and store state. Mainnet is
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/mainnet.mdx]: done

File: docs/docs/networks/goerli.mdx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+- [Prysm](https://docs.prylabs.network/docs/getting-started/)
+- [Teku](https://docs.teku.consensys.net/en/latest/)
+
+### MEV-Boost
+
+We support [mev-boost](https://github.com/flashbots/mev-boost) on mainnet by default, in case you want to remove it,
+run `sedge cli` using the `--no-mev-boost` flag, for example:
+
+```
+sedge cli --no-mev-boost=true
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/goerli.mdx]: done

File: docs/docs/networks/goerli.mdx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Supported Consensus Clients
+- [Lighthouse](https://lighthouse-book.sigmaprime.io/)
+- [Lodestar](https://chainsafe.github.io/lodestar/)
+- [Prysm](https://docs.prylabs.network/docs/getting-started/)
+- [Teku](https://docs.teku.consensys.net/en/latest/)
+
+### MEV-Boost
+
+We support [mev-boost](https://github.com/flashbots/mev-boost) on mainnet by default, in case you want to remove it,
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/goerli.mdx]: done

File: docs/docs/networks/goerli.mdx [hunk 7]
```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 2
+id: goerli
+---
+
+# Goerli
+
+Goerli is an Ethereum test network that allows for blockchain development testing before deployment on Mainnet, the
+main Ethereum network. The Proof-of-Authority test network was established in March 2019. It uses the Clique consensus
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/goerli.mdx]: done

File: docs/docs/networks/gnosis.mdx [hunk 8]
```diff
@@ -0,0 +1,136 @@
+---
+sidebar_position: 4
+id: gnosis
+---
+
+# Gnosis
+
+Gnosis Chain is an EVM based Ethereum sidechain that is designed to be a platform for decentralized prediction markets.
+It is a permissionless network that is governed by the Gnosis community.
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/gnosis.mdx]: done

File: docs/docs/networks/chiado.mdx [hunk 9]
```diff
@@ -1,37 +1,55 @@
...
-sidebar_position: 8
+sidebar_position: 5
 id: chiado
 ---
 
-# Run a Validator or Full Node on Chiado
+# Chiado
+
+Chiado is a Tesnet of Gnosis Chain, that is an EVM based Ethereum sidechain that is designed to be a platform for
+decentralized prediction markets..
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/chiado.mdx]: done

File: docs/docs/networks/goerli.mdx [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+sidebar_position: 2
+id: goerli
+---
+
+# Goerli
+
+Goerli is an Ethereum test network that allows for blockchain development testing before deployment on Mainnet, the
+main Ethereum network. The test network was established in March 2019 with Proof-of-Authority .It uses the Clique consensus
+mechanism and was originally proposed by Chainsafe and Afri Schoedon. Goerli has successfully transitioned to
+Proof-of-Stake following the network’s total block difficulty having crossed 10,079,000 on August 10.
```
Comment (AntiD2ta) [inline docs/docs/networks/goerli.mdx:11]:
```suggestion
Goerli is an Ethereum test network that allows for blockchain development testing before deployment on Mainnet, the
main Ethereum network. The test network was established in March 2019 with Proof-of-Authority. It uses the Clique consensus
mechanism and was originally proposed by Chainsafe and Afri Schoedon. Goerli has successfully transitioned to
Proof-of-Stake following the network’s total block difficulty having crossed 10,079,000 on August 10th, 2022.
```

File: docs/docs/networks/goerli.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (AntiD2ta) [reply docs/docs/networks/goerli.mdx:13]: Still wrong order

File: docs/docs/networks/sepolia.mdx [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+id: sepolia
+---
+
+# Sepolia
+
+Sepolia is a Proof-of-Stake(PoS) testnet where dApp developers can test their applications, and previously, was
+Proof-of-Work(PoW). What differentiates Sepolia from other Ethereum testnets is that most of its PoW validators
+are public validators. Combined with Sepolia being a testnet, this creates the chance that validators will stop
+validating and the chain will fail to finalize, making Sepolia one of the best testnets to see how decentralized
+applications will perform under the harshest conditions.
```
Comment (AntiD2ta) [inline docs/docs/networks/sepolia.mdx:12]: This is referring to the time Sepolia was PoW and you talking using present tense. Update this please and focus on Sepolia being a public network but with permissioned validator sets.

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Requested changes again. Check new comments

File: docs/docs/networks/goerli.mdx (hunk 10 above)
Comment (stdevMac) [reply docs/docs/networks/goerli.mdx]: Solved here [ffde355](https://github.com/NethermindEth/sedge/pull/173/commits/ffde355300b994a40dbc1d6fdae4ede7652ad890)

File: docs/docs/networks/goerli.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (stdevMac) [reply docs/docs/networks/goerli.mdx:19]: Solved here [ffde355](https://github.com/NethermindEth/sedge/pull/173/commits/ffde355300b994a40dbc1d6fdae4ede7652ad890)

File: CHANGELOG.md [hunk 12]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Documentation for all supported networks and clients on each of them.
```
Comment (AntiD2ta) [inline CHANGELOG.md:12]:
```suggestion
- Documentation for all supported networks and clients.
```

File: docs/docs/networks/goerli.mdx [hunk 13]
```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 2
+id: goerli
+---
+
+# Goerli
+
+Goerli is an Ethereum test network that allows for blockchain development testing before deployment on Mainnet, the
+main Ethereum network. The test network was established in March 2019 with Proof-of-Authority. It uses the Clique consensus
```
Comment (AntiD2ta) [inline docs/docs/networks/goerli.mdx:9]:
```suggestion
main Ethereum network. The test network was established in March 2019 with Proof-of-Authority. It used to use the Clique consensus
```

File: docs/docs/networks/sepolia.mdx [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+The Sepolia testnet is a blockchain-based distributed test network that aims to provide users with a safe and secure environment
+for testing and deploying decentralized applications (DApps).
+
+It is an open source platform powered by Cosmos, a blockchain protocol that enables interoperability between different blockchains.
+It enables developers to build, test and deploy DApps on a fully decentralized network.
+
+The Sepolia testnet is a fast, secure, and reliable blockchain network. It is built on Tendermint, a Byzantine Fault Tolerant (BFT) consensus engine
+that enables the network to reach consensus quickly and securely. It also features an optimized Proof-of-Stake consensus mechanism,
+which came in after September 2022’s Ethereum Merge. The Sepolia testnet was designed with scalability and security in mind.
+It can process thousands of transactions per second and is capable of handling a large number of users.
```
Comment (AntiD2ta) [inline docs/docs/networks/sepolia.mdx:17]:
```suggestion
Sepolia is a permissioned Ethereum test network. Along Goerli, it is one of the testnets that is maintained after The Merge. The initial version of Sepolia used a proof-of-work (PoW) consensus mechanism. However, in the light of The Merge, Sepolia followed the mainnet’s example and merged with the proof-of-stake (PoS) Sepolia Beacon Chain. So, Sepolia is now, just like Ethereum, a PoS chain. Furthermore, Sepolia is almost an identical copy of Ethereum, so it accurately mimics the mainnet and supports accurate testing. This enables developers to design, create, test, and track their project’s public performance before going live. 
```

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Three more minor changes to go

**Review by AntiD2ta — APPROVED** [review]
Approved!

**Pull request merged**